roadmap.md

```diff
@@ -76,39 +107,59 @@ state.
    Outcome: Provide more possibilities for Kubernetes releases to be created in
    custom infrastructure environments.
 
-   Enhancement: _none_
+   Enhancement: none
 
-### Known Risks
+1. **Enhance Kubernetes binary artifact management (Consumable)**
 
-1. **We rely on different SIGs for our work**
+   https://github.com/kubernetes/sig-release/issues/1372
 
-   We have a need to discuss our enhancements with different SIGs to get all
-   required information and drive the change. This can lead into helpful, but
-   maybe not expected input and delay the deliverable.
+   Enhancement: _none_
 
-1. **Some topics require initial research**
+   Outcome: Being able to promote files as artifacts and using this mechanism
+   for Kubernetes releases.
 
-   We're not completely aware of all technical aspects for the changes. This
-   means that there is a risk of delaying because of investing more time in
-   pre-research.
+1. **Define and collect metrics about Kubernetes releases (Introspectable)**
 
-### Requests to Other Teams
+   https://github.com/kubernetes/sig-release/issues/1527
 
-1. **SIG K8s Infra**
+   Enhancement: _none_
 
-   For general infrastructure support we rely on.
+   Outcome: Being able to measure and interpret a set of defined metrics about
+   Kubernetes releases to associate actions with those.
 
-1. **SIG Architecture**
+1. **Establish Cluster API as first-class signal for upstream releases
```

Comment:
why specifically Cluster API ?

Reply:
(Also why is this on the SIG Release roadmap and not the SIG Cluster Lifecycle and/or SIG Testing roadmap?)

Reply:
Yeah this one is old. I can remove it, it was never a priority over the past years.